gio/src/auto/socket.rs

```diff
@@ -352,15 +350,6 @@ impl<O: IsA<Socket>> SocketExt for O {
         }
     }
 
-    fn condition_check(&self, condition: glib::IOCondition) -> glib::IOCondition {
```

Comment:
Why does this go away? It is still needed for non-datagram sockets, AFAIU. The two APIs are named the same but not doing exactly the same, right?